### DISCUSSION TITLE
src is not built in source code

### DISCUSSION THREAD

**Discussion by Syberox** [jagrosh/MusicBot#1172] [Questions] [answered]
I downloaded the source code, decided to immediately build src without changes, as a result I caught a bunch of errors, who knows how to build the project, if I didn’t even change it after downloading
![O6p6DBH](https://user-images.githubusercontent.com/74526859/168902939-e30af2b3-8a63-4833-9dbb-440d440199df.png)
.

**Comment by jagrosh** [comment, marked as answer]
Building JMusicBot on your own requires knowledge of Maven for handling dependencies and build instructions. Please learn maven and java on your own, as support for building JMusicBot yourself is not supported.

**Comment by Syberox** [comment]
understandably. One more question. Is there any way to disable the bot
command? So that for example the admin could not use setprefix.

вт, 17 мая 2022 г. в 23:23, John Grosh ***@***.***>:
